network/irc/irc.go

```diff
@@ -596,6 +614,29 @@ func (bot *ircBot) act(theLine *line.Line) {
 		return
 	}
 
+	isAskingForSasl := theLine.User == "" && strings.HasSuffix(theLine.Raw, " CAP * ACK :sasl")
```

Comment:
Something like this would probably be better, and avoide false positives.
```go
isAskingForSasl := strings.toUpper(theLine.Command) == "CAP" && len(theLine.Args) == 2 && strings.toUpper(theLine.Args[1]) == "ACK" && theLine.Content == "sasl"
```